smoke-tests-otel-starter/src/main/resources/application.yaml

```diff
@@ -5,3 +5,6 @@ otel:
         capture-code-attributes: true
   propagators:
     - b3
+  resource:
+    attributes:
+      applicationYaml: "true" # boolean doesn't work here
```

Comment:
Coould you please elaborate this comment?

Reply:
changed to `attributeFromYaml: true # boolean will be automatically converted to string by spring`